README.md

```diff
@@ -11,6 +11,31 @@ Or add to your `requirements.txt`/`setup.py` file the following:
 git+https://github.com/materialscloud/mz-bokeh-package.git@master
 ```
 
-## Changes and Version Number
+To install a branch during development, replace `master` with the branch name.
+This will work for both options above.
+
+### 2. User Authentication
+
+To access the dashboards, user authentication is mandatory and requires setting up of the User API Key as an environment variable:
```

Comment:
This section and the next are not correct (or at least confusing). You have to separate clearly the two cases:
- development - one should set the `API_KEY` environment variable and the `ENVIRONMENT` variable to `dev`. 
- staging/production - the API key needs to be sent via the URL as a query parameter and one needs to add the `--auth-module` flag to the `bokeh serve` command. One can also simulate this locally to test the authentication.